includes/scripts.php

```diff
@@ -300,6 +300,7 @@ function edd_register_admin_scripts() {
 	wp_register_script( 'edd-admin-tax-rates',             $js_dir . 'edd-admin-tax-rates.js',               array( 'wp-backbone', 'jquery-chosen' ), $version, true );
 	wp_register_script( 'edd-admin-email-tags',            $js_dir . 'edd-admin-email-tags.js',              array( 'thickbox', 'wp-util' ), $version );
 	wp_register_script( 'edd-admin-scripts-compatibility', $js_dir . 'edd-admin-backwards-compatibility.js', array( 'jquery', 'edd-admin-scripts' ), $version );
+	wp_register_script( 'edd-admin-tools-export',          $js_dir . 'edd-admin-tools-export.js',            array( 'jquery' ), $version );
```

Comment:
This happens automatically below.

Reply:
Oh perfect, i'll go ahead and remove this then.